procedural/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "openzeppelin-procedural"
```

Comment:
If this is too large of a change at this point feel free to disregard it, but something like `openzeppelin-polkadot-wrappers-proc` would be more consistent I think(given the main package name)

Reply:
It is not large at all :)
Will fix first thing tomorrow